neps/nep-0246.md

```diff
@@ -0,0 +1,550 @@
+---
+NEP: 246 
```

Comment:
Change to 245 to match this PR - also update all references to 245 in this doc.

Reply:
Should this match 245, the issue NEP is 246 and the others seem to reference their issue no ? Just asking that because the PR and issue don't have corresponding numbers 🤔 . Happy to change though just wanted to confirm

Reply:
From a standards perspective, we want to go with the PR number. You have already linked issue 246 in the `DiscussionsTo` section, so I think all of the bases are covered. :)